pages/index.tsx

```diff
@@ -32,9 +33,10 @@ test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (27ms
         <title>Deno - A modern runtime for JavaScript and TypeScript</title>
       </Head>
       <div class="bg-white">
-        <div class="bg-gray-50 border-b border-gray-200">
+        <div class="bg-gray-50 overflow-x-hidden border-b border-gray-200 relative">
+          <Background />
           <Header main />
-          <div class="max-w-screen-sm mx-auto px-4 sm:px-6 md:px-8 pt-12 pb-20 flex flex-col items-center">
+          <div class="z-10 relative max-w-screen-sm mx-auto px-4 sm:px-6 md:px-8 pt-12 pb-20 flex flex-col items-center">
```

Comment:
```suggestion
          <div class="z-9 relative max-w-screen-sm mx-auto px-4 sm:px-6 md:px-8 pt-12 pb-20 flex flex-col items-center">
```

Reply:
Thanks, but unfortunately Tailwind [only has z-index classes](https://tailwindcss.com/docs/z-index) for each 10th value /:

Because it's relative, the z-index wasn't actually needed.

Reply:
That z-9 worked for me tho. Anyway, awesome to hear you fixed it!

Reply:
yep, it worked because that class doesn't exists, so it has the same effect as removing it (: thanks for looking into it tho!